six/modules/c++/six.sidd/wscript

```diff
@@ -1,6 +1,7 @@
 NAME            = 'six.sidd'
 MAINTAINER      = 'adam.sylvester@mdaus.com'
 MODULE_DEPS     = 'scene tiff nitf xml.lite six mem'
+TEST_DEPS       = 'cli six.sicd'
```

Comment:
Rather than doing this and getting this cross thing where SIDD knows about SICD now (even though it's only in the test program), just move this into the modules/c++/samples folder that knows about both SICD and SIDD